packages/headless/src/ssr-commerce.index.ts

```diff
@@ -0,0 +1,222 @@
+import {polyfillCryptoNode} from './api/analytics/analytics-crypto-polyfill';
+
+polyfillCryptoNode();
```

Comment:
Scrap that.
We're gonna put the bar at the right height from the get-go on this package
(i.e. flush those darn polyfill 😂 )